hints/MSWin32.pl

```diff
@@ -1,3 +1,9 @@
 use Config;
-$self->{LIBS} = ['-llibeay32'] if $Config{cc} =~ /cl/;
-$self->{LIBS} = ['-leay32']    if $Config{cc} =~ /gcc/;
+if (my $libs = `pkg-config --libs libcrypto 2>nul`) {
```

Comment:
Should it read `nul` (zero in Danish) or `null` ?

Reply:
It should be `nul` (single `l`) see https://stackoverflow.com/questions/4507312/how-to-redirect-stderr-to-null-in-cmd-exe
It is Windows specific as this hint file is executed only on MS Windows.

Reply:
Thanks for the explanation, I do not use Windows, so it is not familiar to me.